frontend/src/app/actions/experiments.js

```diff
@@ -3,17 +3,18 @@
 import type {
   Experiment,
   UpdateExperimentAction,
-  FetchUserCountsAction
+  FetchUserCountsAction,
+  SetSlugAction
```

Comment:
Since we are rendering the experiments.js at build time in node, we don't synchronously know the slug (until we are running in the browser where we know the full url). So, we need a SetSlugAction to update the state.